src/mute/muteModelTypes.js

```diff
@@ -1,5 +1,23 @@
 // @flow strict-local
+import Immutable from 'immutable';
 
-// No need for Immutable, because -- the server API actually doesn't support
-// incremental changes!  When something changes, it sends the entire new list.
-export type MuteState = Map<number, Set<string>>; // stream ID, topic
+import { type UserTopicVisibilityPolicy } from '../api/modelTypes';
+
+/**
+ * The "visibility policy" our user has chosen for each topic.
+ *
+ * See jsdoc of UserTopicVisibilityPolicyT and related items for background.
```

Comment:
nit: `UserTopicVisibilityPolicy` (without the `T`, happily, thanks to Flow enums!)